ajax/libs/lockr/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lockr",
+  "filename": "lockr.min.js",
+  "version": "0.8.4",
+  "description": "Lockr is an extremely lightweight library (<2kb when minified), designed to facilitate how you interact with localStorage.",
+  "author": {
+    "name": "Dimitris Tsironis",
+    "email": "tsironis.dimitris@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/tsironis/lockr.git"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/tsironis/lockr#readme",
```

Comment:
@x09326 did you manually add this line? I didn't see this from its git repo or npm pkg, there is no need to add homepage link to its git repo.